docs/stencil-docs/localization/multi-language-checkout.md

```diff
@@ -66,11 +66,11 @@ This hidden content includes additional key-value pairs that support translation
 
 ## Adding your own translation values
 
-You can provide values for all of the checkout's supported translation keys (for all the languages you want to support) even without direct access to the hidden parts of the checkout template. Here is how:
+You can provide values for all of the checkout's supported translation keys even without direct access to the hidden parts of the checkout template. Here is how:
 
-1. Download [checkout-en.json](https://github.com/bigcommerce/checkout-js/blob/master/src/app/locale/translations/en.json) (github).
+1. Download checkout's [en.json](https://github.com/bigcommerce/checkout-js/blob/master/src/app/locale/translations/en.json) file (GitHub).
 2. Copy and paste the file's contents into your theme's `en.json` file.
-3. Copy and paste the file's contents the `lang.json` of the other languages you support; for naming requirements, see [Translation Keys](https://developer.bigcommerce.com/stencil-docs/localization/translation-keys#the-schema).
+3. Copy and paste the file's contents into each of the corresponding `json` file for the other languages you support. For naming requirements, see [Translation Keys](https://developer.bigcommerce.com/stencil-docs/localization/translation-keys#the-schema).

```

Comment:
Would change to either:
"each of the corresponding `json` files"
or
"each corresponding `json` file"